### PR TITLE
[Tests] Fix tizen capi unit test

### PR DESCRIPTION
**Pull request by gichan-jang**
Test fails occasionally due to slow speed of arm.
The test was modified to only check if the switch is working.

Signed-off-by: gichan-jang <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: tests/tizen_capi/unittest_tizen_capi.cc [hunk 1]
```diff
@@ -1107,7 +1108,7 @@ TEST (nnstreamer_capi_switch, dummy_01)
   status = ml_pipeline_destroy (handle);
   EXPECT_EQ (status, ML_ERROR_NONE);
 
-  EXPECT_EQ (*count_sink, 3U);
+  EXPECT_GT (*count_sink, 0);
```
Comment (jaeyun-jung) [inline tests/tizen_capi/unittest_tizen_capi.cc:1111]: consider to change EXPECT_LE (*count_sink, 3U);

File: tests/tizen_capi/unittest_tizen_capi.cc [hunk 2]
```diff
@@ -1082,6 +1082,7 @@ TEST (nnstreamer_capi_switch, dummy_01)
   EXPECT_EQ (status, ML_ERROR_NONE);
   EXPECT_TRUE (sinkhandle != NULL);
 
+  EXPECT_EQ (*count_sink, 0);
```
Comment (jaeyun-jung) [inline tests/tizen_capi/unittest_tizen_capi.cc:1085]: IMO this line is unnecessary

File: tests/tizen_capi/unittest_tizen_capi.cc (hunk 1 above)
Comment (gichan-jang) [reply tests/tizen_capi/unittest_tizen_capi.cc:1111]: Good! I will add it. Thank you

File: tests/tizen_capi/unittest_tizen_capi.cc (hunk 2 above)
Comment (gichan-jang) [reply tests/tizen_capi/unittest_tizen_capi.cc:1085]: I will remove it.

File: tests/tizen_capi/unittest_tizen_capi.cc [hunk 3]
```diff
@@ -1107,7 +1107,8 @@ TEST (nnstreamer_capi_switch, dummy_01)
   status = ml_pipeline_destroy (handle);
   EXPECT_EQ (status, ML_ERROR_NONE);
 
-  EXPECT_EQ (*count_sink, 3U);
+  EXPECT_GT (*count_sink, 0U);
+  EXPECT_LE (*count_sink, 3U);
```
Comment (myungjoo) [inline tests/tizen_capi/unittest_tizen_capi.cc:1111]: I'd recommend to add a loop (```while (*count_sink < 3U)```) along with usleep() and timeout-counter.

Comment (gichan-jang) [reply tests/tizen_capi/unittest_tizen_capi.cc]: Thank you! Updated.

File: tests/tizen_capi/unittest_tizen_capi.cc [hunk 4]
```diff
@@ -1093,7 +1093,19 @@ TEST (nnstreamer_capi_switch, dummy_01)
   EXPECT_EQ (status, ML_ERROR_NONE);
   wait_for_start (handle, state, status);
 
-  g_usleep (300000); /* 300ms. Let a few frames flow. */
+  /* Waiting for 3 frames to arrive */
+  while (*count_sink < 3) {
+    timeout_count++;
+    g_usleep (10000);
+    if (timeout_count > 100) {
+      EXPECT_TRUE (false);
```
Comment (gichan-jang) [inline tests/tizen_capi/unittest_tizen_capi.cc:1101]: @myungjoo @jaeyun-jung 
Timeout occurs here in the arm gbs build. Is one second really not enough? Or is there another problem?

**Pull request merged**